static/auth-client.js

```diff
@@ -0,0 +1,9 @@
+import { createAuthClient } from "better-auth/client";
+import { passkeyClient } from "better-auth/client/plugins";
+
+export const authClient = createAuthClient({
+  baseURL: "http://localhost:3000",
+  plugins: [passkeyClient()]
+});
+
+export const { passkey, signIn, signUp } = authClient;
```

Comment:
No newline at end of file suggests that your editor doesn't read the `.editorconfig`.